lib/osf-components/addon/components/registries/review-actions-list/component.ts

```diff
@@ -28,14 +29,13 @@ export default class ReviewActionsList extends Component<Args> {
 
     get latestAction() {
         const { reviewActions } = this;
-        return (reviewActions || [])[0];
+        return A(reviewActions || []).objectAt(0);
     }
 
     @task({ withTestWaiter: true })
     fetchActions = task(function *(this: ReviewActionsList) {
         try {
-            const reviewActions = yield this.args.registration.reviewActions;
-            this.reviewActions = reviewActions.toArray();
```

Comment:
turns out `toArray` does not guarantee order. See https://api.emberjs.com/ember/2.14/classes/Ember.Array/methods/toArray?anchor=toArray